onnxruntime/core/session/ep_library_internal.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/session/ep_library.h"
+#include "core/session/ep_factory_internal.h"
+#include "core/session/onnxruntime_c_api.h"
+#include "core/session/provider_bridge_library.h"
+
+namespace onnxruntime {
+
+struct EpLibraryInternal : EpLibrary {
```

Comment:
you're probably already planning on doing this after addressing more important things. It would be good to have doc comments on these EpLibrary classes to help reader understand differences between them (EpLibraryInternal vs EpLibraryPlugin vs  EpLibraryProviderBridge )